chia/pools/pool_wallet.py

```diff
@@ -718,7 +712,10 @@ async def join_pool(
         if current_state.current.state == LEAVING_POOL:
             history: List[Tuple[uint32, CoinSpend]] = await self.get_spend_history()
             last_height: uint32 = history[-1][0]
-            if self.wallet_state_manager.get_peak().height <= last_height + current_state.current.relative_lock_height:
+            if (
+                self.wallet_state_manager.blockchain.get_peak_height()
```

Comment:
`self.wallet_state_manager.get_peak().height` vs. `self.wallet_state_manager.blockchain.get_peak_height()`

This one might be subtle - is `blockchain.get_peak_height()` more vetted than `get_peak_height()` ?

Reply:
I mean, is it possible for them to be different blocks?

Not a review issue, just wondering.

Reply:
WSM no longer has a `get_peak` method so that code wasn't working before this PR. (I think this was broken in the big merge)

Reply:
cool, thanks